examples/calendar/backend/db.ts

```diff
@@ -0,0 +1,100 @@
+import os from "os";
+import path from "path";
+import { INTEGER, NUMBER, Sequelize, STRING, ENUM } from "sequelize";
+
+import { User, Time, Appointment } from "./models";
+
+const sequelize = new Sequelize("calendar-example", "", undefined, {
+  dialect: "sqlite",
+  storage: path.join(os.tmpdir(), "db.sqlite"),
+  logging: true,
+});
+
+// Initialize all models.
+User.init(
+  {
+    nonce: {
+      allowNull: false,
+      type: INTEGER.UNSIGNED, // SQLITE will use INTEGER
+      defaultValue: (): number => Math.floor(Math.random() * 10000), // Initialize with a random nonce
+    },
+    publicAddress: {
+      // polygon public address of the user
+      allowNull: false,
+      type: STRING,
+      unique: true,
+      validate: { isLowercase: true },
+    },
+    username: {
+      type: STRING, // user defined username
+      unique: true,
+    },
+  },
+  {
+    modelName: "user",
+    sequelize, // This bit is important
+    timestamps: true,
+  }
+);
+
+Time.init(
+  {
+    user: {
+      allowNull: false,
+      type: INTEGER.UNSIGNED, // SQLITE will use INTEGER
+    },
+    status: {
+      allowNull: false,
+      type: STRING,
+      defaultValue: "unavailable", // Initialize with a unavailable status
+    },
+    timestart: {
```

Comment:
it would be better to have lower camel case here.

Reply:
Corrected.